$NetBSD$

The cmake utility recommends using OPENGL_gl_LIBRARY if GLU is not required.
This also fixes the build in some circumstances: if cmake finds an installed
libGLU the build will fail, because we don't buildlink it.

--- src/CMakeLists.txt.orig	2009-09-11 18:03:37.198417478 +1200
+++ src/CMakeLists.txt	2009-09-11 18:03:47.708888770 +1200
@@ -5,7 +5,7 @@
 INCLUDE_DIRECTORIES ( ${ZLIB_INCLUDE_DIR} ${PNG_INCLUDE_DIR}
 		${OPENGL_INCLUDE_DIR}  ${CMAKE_SOURCE_DIR}/include/ )
 LINK_LIBRARIES ( ${CMAKE_DL_LIBS} ${ZLIB_LIBRARIES} ${PNG_LIBRARY}
-		${OPENGL_LIBRARIES} )
+		${OPENGL_gl_LIBRARY} )
 SET ( SRCS ${APPNAME}.c
 )
 SET(CMAKE_SHARED_LIBRARY_C_FLAGS "-fPIC")
